consensus/module.go

```diff
@@ -236,15 +238,15 @@ func (m *consensusModule) SetBus(pocketBus modules.Bus) {
 	}
 }
 
-func (*consensusModule) ValidateGenesis(genesis *genesis.GenesisState) error {
+func (*consensusModule) ValidateGenesis(gen *genesis.GenesisState) error {
 	// Sort the validators by their generic param (i.e. service URL)
-	vals := genesis.GetValidators()
+	vals := gen.GetValidators()
 	sort.Slice(vals, func(i, j int) bool {
 		return vals[i].GetGenericParam() < vals[j].GetGenericParam()
 	})
 
 	// Sort the validators by their address
-	vals2 := vals[:]
+	vals2 := vals
```

Comment:
You will likely need to add a nolint rule here as well.

```suggestion
        // HACK: We're making a copy of `vals` to make sure that the ordering (see comment below) is enforced.
	vals2 := vals[:]
```